init_db.sh

```diff
@@ -26,6 +26,6 @@ echo 'Installing app dependencies...'
 pip3 install -r app/requirements.txt >/dev/null
 
 echo 'Initializing database...'
-cd app/
+cd app/ || exit 1
```

Comment:
You can drop the `/` in `app/` to make it similar to other similar statements in other scripts we have.